.github/workflows/CI.yml

```diff
@@ -145,9 +145,10 @@ jobs:
           docker run --volume ${{ github.workspace }}:/app --workdir /app --privileged builder sh -c "${{ matrix.docker_cmd }}"
 
       - name: Upload artifacts
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         with:
           path: ./prebuilds
+          name: prebuilds-${{ github.run_number }}-${{ github.run_attempt }}-${{ strategy.job-index }}
```

Comment:
Can't we merge these prebuilds into one zip? This makes it harder to download them and make a release.

Reply:
I figured these were artifacts for diagnostic purposes only, not for release. Yes, it's a great idea to merge these and I'll do so.